### ISSUE TITLE
`VfsSuperBlock` in libgum

### ISSUE THREAD

**Issue by zilinc** [NICTA/cogent#227]
The type `VfsSuperBlock` is not defined in any .cogent file, and is not used in any filesystem(?).

**Comment by ajaysusarla** [comment]
That is true, we don't use it. My idea was to eventually use all definitions from libgum when I started with `skelfs`. But I hadn't got that stage yet. And with our effort to 'design right', I'm not sure how it fits in yet.
I would like to retain in there for now, if that isn't a problem.

**Comment by zilinc** [comment]
We don't have to fix it right now. I just wanted to know how things worked in order to make `make test-libgum` pass.
